DataFormats/L1TParticleFlow/interface/gt_datatypes.h

```diff
@@ -169,6 +169,8 @@ namespace l1gt {
       return ret;
     }
 
+    inline static Sum unpack(const uint64_t &src) { return unpack_ap(src); }
```

Comment:
at some point people might complain that the naming is inconsistent since `pack` returns with the ap type and not `uint64_t`, and so we might want to have `pack_ap`/`unpack_ap`, or have both unpack methods called `unpack` (relying on overloading to pick the right one) if the cast from `ap_uint<64>` to `uint64_t` is properly implemented so that`uint64_t x = obj.pack();`  works